core/src/edu/rice/rbox/Game/Common/Utils/CoinStateMessage.java

```diff
@@ -1,16 +1,16 @@
-package edu.rice.rbox.Game.Common.Utils;
-
-import network.GameNetworkProto.CoinMessage;
-
-public class CoinStateMessage {
```

Comment:
All of these files that use the generated classes from grpc should all work in the main branch -- none of them should be commented out.